src/main/java/com/technocredits/orangeHRM/base/PredefinedMethods.java

```diff
@@ -0,0 +1,38 @@
+package com.technocredits.orangeHRM.base;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+
+//Selenium Wrapper Class
+public class PredefinedMethods {
+	WebDriver driver=null;
+
+	public void initialization() {
+		System.setProperty("webdriver.chrome.driver",
+				"C://chetan.phadke//Backup//Data from D//software//chromedriver_win32//chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
```

Comment:
```suggestion
		WebDriver driver = new ChromeDriver();
		
		Earlier you already given WebDriver driver = null and again here you are giving return type of driver Why ?
```

Reply:
Okay. My bad. Deleted the return type webdriver.